src/PeriodCollection.php

```diff
@@ -144,6 +144,21 @@ public function isEmpty(): bool
         return count($this->periods) === 0;
     }
 
+    public function subtract(PeriodCollection|Period $periods)
```

Comment:
I've been trying to apply the naming convention to call these other periods `$others` and `$other` in all methods, could you change the variable names?